requirements/doc.txt

```diff
@@ -1,17 +1,21 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.9
 # by the following command:
 #
 #    make upgrade
 #
 accessible-pygments==0.0.4
     # via pydata-sphinx-theme
-alabaster==0.7.13
+alabaster==0.7.16
```

Comment:
This version requires `python>=3.9`. Please re-create your `venv` with 3.8, or if this change is recommended, please change the CI pipeline to test against python3.9.

Reply:
Sorry about that.. I reverted to python3.8 and re-ran `make upgrade`, see [fc0ce5e](https://github.com/openedx/platform-plugin-aspects/pull/7/commits/fc0ce5e0e842f342de17314568114b39cefcd91c).